modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/dashboard-collection/common/components/container/index.js

```diff
@@ -28,10 +34,12 @@ function Container({children, className, footer, title}) {
 				{children}
 
 				{footer && (
-					<div>
+					<div className={`${positionButton ? 'd-flex' : ''}`}>
```

Comment:
```suggestion
					<div className={classnames({
					  positionButton: 'd-flex'
					})}>
```

Eh interessante usarmos a lib do classnames.

Reply:
adicionado lib classNames